packages/web-components/fast-components/src/styles/density.ts

```diff
@@ -0,0 +1,249 @@
+import { composedParent, DesignToken } from "@microsoft/fast-foundation";
+import { designUnit, strokeWidth } from "../design-tokens";
+
+const { create } = DesignToken;
+
+/**
+ * A Design Token representing the visual height of a single line of `base`-sized text.
+ *
+ * @remarks
+ * A variant on the type ramp for the common usage of a single line of text as in most components, adjusted for visual size rather than leading between multiple lines.
+ *
+ * For text-only and icon-only buttons to be the same height, this value should be the same as the icon size.
+ *
+ * @defaultValue 16px
+ *
+ * @public
+ */
+export const typeRampBaseVisualHeight = create<string>(
```

Comment:
I considered other options for this, but I think however this ends up working there does need to be separate values for handling single vs multiple lines of text. I think the stronger choice is actually something like:

single visual line height (this value) + leading (new property) = multi-line height (existing property)

This is more classically the way text layout is thought of and opens up increased possibilities including https://medium.com/microsoft-design/leading-trim-the-future-of-digital-typesetting-d082d84b202 and https://seek-oss.github.io/capsize/. See the RFC for why text is so hard to size correctly.